packages/postgres-database/src/simcore_postgres_database/migration/versions/967cff1d52ac_new_resource_tracker_container_table.py

```diff
@@ -0,0 +1,49 @@
+"""New resource_tracker_container table
+
+Revision ID: 967cff1d52ac
+Revises: c8f072c72adc
+Create Date: 2023-06-13 06:37:31.542190+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "967cff1d52ac"
+down_revision = "c8f072c72adc"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "resource_tracker_container",
+        sa.Column("id", sa.String(), nullable=False),
+        sa.Column("image", sa.String(), nullable=False),
```

Comment:
What is the image? I would have here key and version of the service, since you track the resource usage of user services and not of the platform services.
I think this will help us with the queries as well.

Reply:
image is one-to-one field that i get from the container labels provided by prometheus